internal/venice-common/src/main/java/com/linkedin/venice/writer/VeniceWriter.java

```diff
@@ -516,12 +498,8 @@ public Future<PubSubProduceResult> delete(K key, PubSubProducerCallback callback
    *                         >=0: Leader replica consumes a delete message from real-time topic, VeniceWriter in leader
    *                              is sending this message to version topic with extra info: offset in the real-time topic.
    * @param logicalTs - An timestamp field to indicate when this record was produced from apps point of view.
-   * @param deleteMetadata - a DeleteMetadata containing replication metadata related fields (can be null).
```

Comment:
Is it intentional to delete the param JavaDoc?

Reply:
No, it's not. Thanks for catching it.